src/test/java/com/google/api/generator/gapic/composer/goldens/IdentityClient.golden

```diff
@@ -53,6 +53,14 @@ import javax.annotation.Generated;
  *
  * <p>To customize credentials:
  *
+ * <pre>{@code
+ * IdentitySettings identitySettings =
+ *     IdentitySettings.newBuilder()
+ *         .setCredentialsProvider(FixedCredentialsProvider.create("myCredentials"))
```

Comment:
Just noticed this - this should be some variable, not a string. Example [here](https://github.com/googleapis/java-dataproc/blob/master/google-cloud-dataproc/src/main/java/com/google/cloud/dataproc/v1/AutoscalingPolicyServiceClient.java#L87)